Wire-iOS/Sources/Helpers/URL+Wire.swift

```diff
@@ -197,7 +193,7 @@ extension URL {
     }
 
     static var wr_wireEnterpriseLearnMore: URL {
```

Comment:
Since this is about the pricing now maybe a more appropriate name would be something like:

```suggestion
    static var wr_wireEnterprisePricing: URL {
```

Reply:
I'd like to keep this link name, because the web page contains pricing info, `Wire for Enterprise` and `Wire for Free `description. Previously, there were different links, and they were used for different purposes

Reply:
👍